### DISCUSSION TITLE
Use same Private Keys without new handlers

### DISCUSSION THREAD

**Discussion by sarsey-walker** [eddieoz/openxrypt#11] [Ideas]
At "Manage Private Keys", if the user want to use the same private key for more than one web site/account.

The the idea is to have a checkbox, or if the user not set a handler or number, save without it, maybe generating a UUID. 

When encrypt and decrypt, check if the sender user have an ID for the Website saved, if not, try find this UUID.

**Comment by eddieoz** [comment]
I think we could put a new button named 'Add New Handle' under 'Show Pubkey'. And instead of creating a new UUID and a logic like relational dbs, it could be added to an array like ['handle1', 'handle2',...] for the user of that private key.
By using the same structure and registering users in arrays instead of a strings, we can do the same for the public keys also. What you think about it?

**Comment by sarsey-walker** [comment]
Thats a good idea. Would be more simple.
